src/components/Banner/index.tsx

```diff
@@ -28,8 +25,53 @@ const Banner = ({
         </a>
         {text}
       </p>
-    </div>
-  ) : null;
+    );
+  }
+
+  return null;
+};
+
+const useHtmlContent = ({ html, link }: BannersIndex) => {
+  if (html) {
+    // @note this is a workaround and it would be better if nodejs.org
+    // repository would provide absolute urls instead of relative for us
+    const replaceLocalLinksToNodejsOrg = html.replace(
+      "'/static/",
+      "'http://nodejs.org/static/"
+    );
+
+    return (
+      <a
+        href={isAbsoluteUrl(link) ? link : `http://nodejs.org/${link}`}
+        target="_blank"
+        rel="noopener noreferrer"
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{ __html: replaceLocalLinksToNodejsOrg }}
```

Comment:
Can you import and use dompurify here.
`const sanitizer = dompurify.sanitize;`

`dangerouslySetInnerHTML={{ __html: sanitizer(replaceLocalLinksToNodejsOrg) }}`

Reply:
Done!